src/index.ts

```diff
@@ -1,20 +1,45 @@
-import express from 'express';
+import 'reflect-metadata';
+import express, { NextFunction, Request, Response } from 'express'
 import 'dotenv/config';
 import cors from 'cors';
-import 'reflect-metadata';
+import AppError from './error/appError';
+import { Router } from 'express';
+import brandRouter from './modules/BrandEntity/router/brand.routes';
+import modelRouter from './modules/ModelEntity/router/model.routes';
+import versionRouter from './modules/VersionEntity/router/Version.routes';
+
 
 const port = process.env.PORT || 9000;
 const hostname = process.env.HOSTNAME || 'http://localhost';
 
 const app = express();
 app.use(express.json());
 app.use(express.urlencoded({extended: true}));
+
 app.use(cors({
     origin: ['http://localhost:9000']
```

Comment:
use the constant create at line 12 in replacement to: origin: [`http://localhost:${port}`]